LoRaEngine/modules/LoRaWanNetworkSrvModule/LoraTools/LoRaMessage/LoraMessageWrapper.cs

```diff
@@ -65,33 +65,58 @@ public LoRaMessageWrapper(byte[] inputMessage, bool server = false, string AppKe
                 {
                     Logger.Log($"DataUp {payload}", Logger.LoggingLevel.Full);
 
+                    // Deserialized for uplink messages
                     var payloadObject = JsonConvert.DeserializeObject<UplinkPktFwdMessage>(payload);
                     PktFwdPayload = payloadObject;
                     // set up the parts of the raw message   
                     // status message
-                    if (PktFwdPayload.GetPktFwdMessage().Rxpks[0].data != null)
+                    if (PktFwdPayload != null)

```

Comment:
@Mandur can you review this part view that it is in your code part

Reply:
This seems good to me